docs/source/api/tutorials/In Depth 3 - Drawing DiHypergraphs.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "ea8aad6a",
```

Comment:
<p>This case looks pretty bad now. How is it different from before? Should we not display it if it looks that bad?</p><p> </p><p>Also I thought you mentioned to change the pos into a single tuple argument, but maybe it's just the notebook that's not up to date, I haven't checked everything yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
This is now removed and the `pos` issue is fixed.